src/input-field.tsx

```diff
@@ -43,7 +43,7 @@ const Label = styled.label({
 export interface InputProps
     extends Omit<
             React.HTMLProps<HTMLInputElement>,
-            'name' | 'height' | 'width' | 'wrap' | 'label' | 'onResize' | 'onResizeCapture'
+            'name' | 'height' | 'width' | 'wrap' | 'label' | 'placeholder' | 'onResize' | 'onResizeCapture'
```

Comment:
`React.HTMLProps<HTMLInputElement>` defines placeholder as `string | undefined`, which is what we're doing right?  don't think you need to omit it and define it below